src/main/java/com/jfrog/ide/idea/ui/webview/WebviewObjectConverter.java

```diff
@@ -30,15 +30,14 @@ public static DependencyPage convertIssueToDepPage(IssueNode issueNode) {
         if (issueNode.isApplicable() != null) {
             if (issueNode.isApplicable()) {
                 List<ApplicableIssueNode> applicableIssues = issueNode.getApplicableIssues();
-                if (applicableIssues.size() > 0) {
-                    Evidence[] evidences = new Evidence[applicableIssues.size()];
-                    int i = 0;
-                    for (ApplicableIssueNode applicableIssue : applicableIssues) {
-                        evidences[i] = new Evidence(applicableIssue.getReason(), applicableIssue.getFilePath(), applicableIssue.getLineSnippet());
-                        i++;
-                    }
-                    applicableDetails = new ApplicableDetails(true, evidences, null);
+                String searchTarget = applicableIssues.get(0).getScannerSearchTarget();
+                Evidence[] evidences = new Evidence[applicableIssues.size()];
+                int i = 0;
+                for (ApplicableIssueNode applicableIssue : applicableIssues) {
+                    evidences[i++] = new Evidence(applicableIssue.getReason(), applicableIssue.getFilePath(), applicableIssue.getLineSnippet());
                 }
+                applicableDetails = new ApplicableDetails(true, evidences, searchTarget);
+
```

Comment:
This logic should be extracted into a dedicated function (or class?). Also, writing tests for this logic will be easier.